agogosml_cli/Makefile

```diff
@@ -76,6 +76,9 @@ test-dev:
 test-all: ## run tests on every Python version with tox
 	tox
 
+test-py37: ## run tests on specific Python version with tox
+	tox -e py37
```

Comment:
You can avoid hard-coding py37 here by using something like this:

```makefile
TOX_VERSION=py37

test-single: ## run tests on specific Python version with tox
	tox -e ${TOX_VERSION}
```

You can then call the recipe like so and it will default to py37:

```sh
make test-single
```

Alternatively you can also inject the `TOX_VERSION` in two ways:

```sh
# approach 1:
export TOX_VERSION=py36
make test-single

# approach 2:
make test-single -e TOX_VERSION=py36
```

Reply:
+1 to `make test-single`